armi/reactor/grids.py

```diff
@@ -1154,28 +1159,30 @@ def getMinimumRings(self, n):
         """
         Return the minimum number of rings needed to fit ``n`` objects.
         """
-        return self._getMinimumRingsStatic(n, throughCenter=self._isThroughCenter())
-
-    @staticmethod
-    def _getMinimumRingsStatic(n, throughCenter):
-        # Keep private because this is really here to support old locations stuff that
-        # hasn't been removed yet. See module docs for locations.py for more details.
```

Comment:
We still havent solved the problem that requires this to be here. Again, the Blocks don't necessarily have a grid. If they did have a grid, at least it would let us express the appropriate even/odd through center behavior (the locations.py logic that the geometry converters was using just assumes `True` 😲). One way to get rid of this would be to address some of the stuff that I talk about up there.